pyproject.toml

```diff
@@ -20,6 +20,7 @@ numpy = "^1.21"
 pydantic = "^2.5"
 lxml = "^4.6"
 meshkernel = "^4.0.2"
+strenum = "^0.4.15"
```

Comment:
This is the latest version, please consider loosening this constraint to avoid dependency conflicts when combining it with other packages in an environment.

Reply:
Updated version requirement for strenum to ^0: >= 0.0.0, <1.0.0